### ISSUE TITLE
experience bar breaks in party member list

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitrpg-ios#1185]
> A party members exp bar is longer than the bar. This probably should not happen anyway, but the view should handle this with a max length anyway.
![image0](https://user-images.githubusercontent.com/102926641/164986650-66c8916a-70df-4025-aa98-9d44c72c7b9a.jpeg)

**Comment by saraolson** [comment]
this happens when someone in the party finishes a quest and gets more than their max xp before leveling (which is a longstanding issue). the exp bar should have a max set so even when the number goes beyond the max, the bar doesn't keep growing

**Comment by dan-e-miller** [comment]
Hi @saraolson,

I would like to work on this issue.

Thanks!
Dan

**Comment by dan-e-miller** [comment]
I just went ahead and posted a PR for this issue.

@saraolson, I'm running into issues with the Continuous Integration tests. The change I posted fixes the issue and is very simple, using syntax and formatting already used in multiple other places in the app. Any chance there could be issues with the tests themselves or the CI environment? Otherwise I'm happy to make any updates that are needed for the tests to pass.
